### ISSUE TITLE
Add a new `Tabled` attribute `format = ""` as a shortcut for a `display_with`

### ISSUE THREAD

**Issue by zhiburt** [zhiburt/tabled#396]
I guess it's make sense to do.
To reduce a boilerplate for common simple cases.

```rust
#[derive(Tabled)]
struct Motorcycle {
  weight: usize,
  #[tabled(format = "{} cc")]
  cc: usize,
}
```

I think a 2nd version could exist, to being able to use a self as argument.

```rust
#[derive(Tabled)]
struct Motorcycle {
  weight: usize,
  #[tabled(format("{}/{} cc/kg", self.cc, self.weight))]
  cc: usize,
}
```

**Comment by thomassimmer** [comment]
Hello,

I started working on this. Just need to make tests work for enums. Could you assign this to me ?

Cheers,
Thomas

**Comment by zhiburt** [comment]
@thomassimmer 

Thanks

If you have any question please ping me.

But overall it shall be very very very similar to `display_with` implementation.
Don't be frightened by quite bad quality code at `tabled_derive/src/lib.rs`.

Take care.

**Comment by zhiburt** [comment]
closed by #398
